pandas/tools/util.py

```diff
@@ -238,3 +238,39 @@ def to_numeric(arg, errors='raise', downcast=None):
         return values[0]
     else:
         return values
+
+
```

Comment:
Thanks for this.  Just to make sure I understand everything, this is better than the original solution, because it doesn't depend upon the object passed to `concat` yet still uses the `DataFrame` method without having to reproduce it (and add unnecessary duplication)?  And there isn't any problem with creating an empty `DataFrame` instance in order to use the method because we can consistently create it every time?

This is my first PR to Pandas, so I appreciate the help!